cirq/circuits/circuit_test.py

```diff
@@ -219,10 +219,20 @@ def test_empty_moments():
     cirq.testing.assert_has_diagram(circuit,
                                     "a: ───X───X───────X───",
                                     use_unicode_characters=True)
+    cirq.testing.assert_has_diagram(circuit,
+            '\n'.join(['a', '│', 'X', '│', 'X', '│', '│', '│', 'X', '│']),
```

Comment:
Please render the diagram directly using triple quotes so it's clearer what it looks like.